pkg/storage/module/delete_test.go

```diff
@@ -0,0 +1,55 @@
+package module
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/gobuffalo/envy"
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/stretchr/testify/suite"
+)
+
+type DeleteTests struct {
+	suite.Suite
+}
+
+func TestDelete(t *testing.T) {
+	suite.Run(t, new(DeleteTests))
+}
+
+func (m *DeleteTests) TestDeleteTimeout() {
+	r := m.Require()
+	envy.Set("ATHENS_TIMEOUT", "1")
+	defer envy.Set("ATHENS_TIMEOUT", "300")
+	err := Delete(context.Background(), "mx", "1.1.1", delWithTimeout)
+
+	me := err.(*multierror.Error)
```

Comment:
I'm not sure if you need to cast to a `multierror.Error` here. I think the method `.Error` will return the full error string of all errors collected. Maybe I'm confused, but the error returned from delete is a multierror already? Or does it lose those that type after being returned.

Reply:
This one is more a question.

Reply:
You're right - the cast isn't needed.

Reply:
Delete returns an `error` and I want to check the no. of errors so I call `WrappedErrors()`